contingency/contingency-api/src/main/java/com/powsybl/contingency/Contingency.java

```diff
@@ -32,12 +31,12 @@
 
     public Contingency(String id, ContingencyElement... elements) {
         this.id = Objects.requireNonNull(id);
-        this.elements = ImmutableList.copyOf(elements);
+        this.elements = Arrays.asList(elements);
     }
 
     public Contingency(String id, List<ContingencyElement> elements) {
         this.id = Objects.requireNonNull(id);
-        this.elements = ImmutableList.copyOf(elements);
+        this.elements = new ArrayList<>(Objects.requireNonNull(elements));
```

Comment:
Fix the `getElements()` method: the list should not be modifiable, except using `addElement` and `removeElement`